python_modules/dagster/dagster_tests/core_tests/test_event_logging.py

```diff
@@ -76,24 +86,24 @@ def _event_callback(record):
     assert result.success
     assert events
 
-    start_event = single_event(events, EventType.EXECUTION_PLAN_STEP_START)
+    start_event = single_dagster_event(events, ExecutionStepEventType.STEP_START)
     assert start_event.pipeline_name == 'single_solid_pipeline'
-    assert start_event.solid_name == 'solid_one'
+    assert start_event.dagster_event.solid_name == 'solid_one'
+    assert start_event.dagster_event.solid_definition_name == 'solid_one'
```

Comment:
Soonish in a follow on diff I think we should have a top level event that is either typed LogMessage or DagsterEvent with a small shared interface. e.g. (LogMessage, DagsterEvent)